gui/wxpython/docs/wxgui_sphinx/conf.py

```diff
@@ -10,22 +10,14 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys
-import os
 from datetime import date
 import string
 from shutil import copy
+from grass.script import core
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
```

Comment:
The comment should go with that. It does not fit much with the code and what is relevant is likely in Sphinx doc.